cmd/codegen/generator/generator.go

```diff
@@ -12,7 +12,7 @@ import (
 
 	"dagger.io/dagger"
 	"github.com/dagger/dagger/cmd/codegen/introspection"
-	"github.com/dagger/dagger/core/modules"
+	"github.com/dagger/dagger/core"
```

Comment:
Huh we should avoid depending on this. `core` is a huge module, I think if we can remove the dependency we should hopefully make the binary a lot smaller (it's only for `core.ModuleSourceKind`)